swagger.yaml

```diff
@@ -2716,6 +2716,55 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/operators/bundles:
+    get:
+      tags:
+        - operators
+      summary: Get list of bundle names
+      description: Retrieves a list of bundle names.
+      operationId: V2GetBundles
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/bundle'
+        "500":
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/error'
+
+  /v2/operators/bundles/{bundle_name}:
+    get:
+      tags:
+        - operators
+      summary: Get operator properties for a bundle
+      description: Retrieves an array of operator properties for the specified bundle.
+      operationId: V2GetBundleOperators
+      parameters:
+        - in: path
+          name: bundle_name
+          description: The name of the bundle.
+          required: true
+          type: string
+          enum: ['virtualization', 'openshiftai']
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              type: string
```

Comment:
This leaves little room for expanding the definitions of bundles without breaking backwards compatibility. Can we instead the define the new `bundle` type as an object. It can then have a field that contains the names of the operators, but we can also extend it in the future if needed.